src/pages/edit_resource/contributors_step/ContributorFields.tsx

```diff
@@ -42,12 +42,25 @@ const StyledFieldsWrapper = styled.div`
   }
 `;
 
-const StyledTextField = styled(TextField)`
-  margin-right: 1rem;
+interface TextFieldSizerProps {
+  isCurator: boolean;
+  id: string;
+  className?: string;
+}
+const TextFieldSizer: FC<TextFieldSizerProps> = ({ children, className, id }) => {
```

Comment:
Skjønte ikke nytten til komponenten her ? er det for å få in om kurator er satt ? Om det er det kan det forenkles med å legge til props rett på styledcomponent-elementet

Reply:
Supert, da lærte jeg noe nytt.